docs/pages/pack/docs/features/css.mdx

```diff
@@ -50,6 +52,12 @@ Using the CSS `@import` syntax to import other CSS files works **out-of-the-box*
 @import './dark.css';
 ```
 
+<Callout type="info">
+  Turbopack does not currently support absolute uri imports in css, such as `@import(https://example.com/stylesheet.css)`. As a workaround, add
+  `<link rel="stylesheet" href="https://example.com/stylesheet.css"></link>` to your HTML document. Next.js users using `pages/` should
+  [add this to `pages/_document.js`](https://nextjs.org/docs/messages/no-page-custom-font#possible-ways-to-fix-it).
```

Comment:
Next.js 13 also has `app/layout.js`: https://beta.nextjs.org/docs/routing/pages-and-layouts#root-layout-required

I'm not sure whether we support this yet.